RELEASE-NOTES.txt

```diff
@@ -3,6 +3,9 @@
 17.8
 -----
 * [*] Fixed a bug where the web version of the editor did not load when using an account created before December 2018. [#14762]
+* [*] Block editor: Update loading and failed screens for web version of the editor [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3573]
+* [*] Block editor: Handle floating keyboard case - Fix issue with the block selector on iPad. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3687]
```

Comment:
This entry shouldn't be included in WPAndroid as it's related to a fix that only applies to iOS, I've created [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/15017) to update it.